core/src/transfer/data.rs

```diff
@@ -79,6 +124,22 @@ pub struct ContractDeploy {
     pub nonce: u64,
 }
 
+/// Data for used to identify a BLOB data
```

Comment:
```suggestion
/// Data used to identify a BLOB data
```
Also, not sure if BLOB (capitalized) has a special meaning. If not, I would keep it as `blob`